pkg/outputter.go

```diff
@@ -56,6 +57,9 @@ func (p *FileOutputStreamProvider) GetWriter(ctx context.Context, iface *Interfa
 		relativePath := strings.TrimPrefix(
 			filepath.Join(filepath.Dir(iface.FileName), p.filename(caseName)),
 			absOriginalDir)
+
+		relativePath = strings.Replace(relativePath, "/internal/", "/_internal/", -1)
```

Comment:
Would it work with `/internal_/` too? If yes, I would prefer to keep the lexical sorting in my file explorer intact as much as possible by having the `_` at the end